assets/src/edit-story/components/library/panes/text/textPane.js

```diff
@@ -73,7 +111,46 @@ function TextPane(props) {
       </Section>
       {showTextSets && (
         <Section title={__('Text Sets', 'web-stories')}>
-          <SectionContent>{__('Coming soon.', 'web-stories')}</SectionContent>
+          <TextSetContainer>
+            <UnitsProvider
+              pageSize={{
+                width: ITEM_SIZE,
+                height: ITEM_SIZE,
+              }}
+            >
+              {textSets.map((elements, index) => (
+                <TextSetItem
+                  key={index}
+                  style={{ justifySelf: index % 2 === 0 ? 'start' : 'end' }}
+                  onClick={() => insertElements(elements)}
+                >
+                  {elements.map(
+                    ({
+                      id,
+                      content,
+                      x,
+                      y,
+                      textSetWidth,
+                      textSetHeight,
+                      ...rest
+                    }) => (
+                      <DisplayElement
+                        previewMode
+                        key={id}
+                        element={{
+                          id,
+                          content: `<span style="color: #fff">${content}<span>`,
+                          x: x + (PAGE_WIDTH - textSetWidth) / 2,
+                          y: y + (PAGE_HEIGHT - textSetHeight) / 2,
+                          ...rest,
```

Comment:
This works since it is not a generated asset or static SVG. An alternative could be HTML but the DisplayElement is a 1:1 mapping in how it would show in the editor.

Reply:
Yeah, leveraging the DisplayElement control lets us position and size it with respect to it's container div and it contains all the logic to properly render every element scaled too, which is nice.  But we'll definitely have to figure out a better way for adjusting the colors.  Right now I'm just making all text white (which won't work for shapes), but I think there's a pretty clean way to adjust the color by element type.